.github/workflows/publish.yml

```diff
@@ -30,6 +30,9 @@ jobs:
     - name: Install dependencies in react-components
       run: yarn install
       working-directory: packages/react-components
+    - name: Install dependencies in css-library
+      run: yarn install
+      working-directory: packages/css-library
```

Comment:
I'm not sure how to test this without creating a `core` release 🤔

Reply:
not sure...but the fact that you can test that the change gets the workflow past the failure seems good enough to merge